pkg/operator/controller/ingress/status_test.go

```diff
@@ -877,6 +904,12 @@ func Test_computeLoadBalancerProgressingStatus(t *testing.T) {
 			platformStatus: awsPlatformStatus,
 			expectStatus:   operatorv1.ConditionFalse,
 		},
+		// TODO: openstack tests
+		{
+			name:         "LoadBalancerService, OpenStack floating IP spec and status are empty",
+			ic:           loadBalancerIngressControllerWithFloatingIP("", ""),
+			expectStatus: operatorv1.ConditionTrue,
```

Comment:
I'm a bit confused why this is True. Shouldn't progressing be False if both spec and status are empty?

Also, we should add more test cases here, like when it's not empty, but that's probably what you mean by the TODO.

Reply:
A bad test. Fixing it and adding more of course.